rclpy/rclpy/qos.py

```diff
@@ -316,7 +327,10 @@ class QoSLivelinessPolicy(QoSPolicyEnum):
     MANUAL_BY_NODE = RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE
     RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC = 3
     MANUAL_BY_TOPIC = RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC
-
+    
+# Naming the class DurabilityPolicy QoSDurabilityPolicy 
+# because it is used in other places with the "Qos" infront
+QoSDurabilityPolicy=DurabilityPolicy
```

Comment:
Did you mean `LivelinessPolicy` here? It looks like `DurabilityPolicy` is above, and the class name for `QoSLivelinessPolicy` should be changed as well.